application/controllers/ListController.php

```diff
@@ -29,6 +31,14 @@ public function init()
 
     public function hostsAction()
     {
+        if (Module::exists('icingadb') && IcingadbSupport::useIcingaDbAsBackend()) {
+            if ($hostName = $this->params->shift('host')) {
+                $this->params->set('name', $hostName);
```

Comment:
Please use `host.name` here. Just `name` may work fine, but that's more of a convenience for lazy users typing this into the search bar :wink: